docs/docs/http-api.mdx

```diff
@@ -50,54 +50,44 @@ The [SocketIO channel](./connectors/your-own-website.mdx#websocket-channel) does
 
 ## Security Considerations
 
-We recommend to not expose the Rasa Server to the outside world, but
-rather connect to it from your backend over a private connection (e.g.
-between docker containers).
+We recommend that you don't expose the Rasa Server to the outside world directly, but
+rather connect to it via e.g. Nginx.
 
 Nevertheless, there are two authentication methods built in:
 
 ### Token Based Auth
 
-Pass in the token using `--auth-token thisismysecret` when starting
+To use a plaintext token to secure your server, specify the token in the argument `--auth-token thisismysecret` when starting
```

Comment:
These are great additions, however I just realised that I didn't clarify in my ticket that the page that is missing instructions is the [API spec page ](https://rasa.com/docs/rasa/pages/http-api)🙈

Reply:
ahhh I see I see, will further update the spec

Reply:
I added some description to the spec page, lmk if it serves the purpose you had in mind

Reply:
Looks great 🙌🏼